Maquetado/scss/styles.scss

```diff
@@ -0,0 +1,173 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: 24px;
+  justify-content: flex-end;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.techno-title {
+  font-size: 12px;
+}
+
+.type-tool {
+  font-size: 24px;
+  position: relative;
+  transform: translateX(100%);
+
+  &::before {
+    background-image: url('assets/triangle.svg');
+    background-repeat: no-repeat;
+    content: '';
+    height: 90px;
+    position: absolute;
+    transform: translateX(-50%);
+    top: -35px;
```

Comment:
Avoid negative values, you can do `bottom: 35px;` and should have the same result :smile: